.github/workflows/iotlab-ci.yml

```diff
@@ -0,0 +1,119 @@
+name: iotlab-ci
```

Comment:
Also, I'm not sure about the name... how about (not sure I like that either, but it is more descriptive...)
```suggestion
name: compile-and-test-in-iotlab
```

Reply:
Its just: The current name sticks out as very non-descriptive

![image](https://user-images.githubusercontent.com/675644/89015666-b570d280-d317-11ea-94b3-b7f3c00d8545.png)

Reply:
My personal preference goes for the original name: it's descriptive enough (CI on IoT-LAB) and short.

Reply:
>  it's descriptive enough (CI on IoT-LAB) and short.

What is CI on `iotlab`? The release tests are also CI. Running a specific on `iotlab-m3` is also `iotlab-ci`. You see that it is not very specific?

Reply:
> What is CI on iotlab? 

Running the tests on boards provided by IoT-LAB. As usual, we can nitpicks on words and loose our time. I choose this names for a good reason, I prefer to keep it.

Reply:
> Running the tests on boards provided by IoT-LAB. […]

Then maybe `test-on-iotlab`?

> As usual, we can nitpicks on words and loose our time. I choose this names for a good reason, I prefer to keep it.

I don't think I am nitpicking in this instance. Names are important, especially if they are shown very prominently. I don't think the name `iotlab-ci` is not very good.

Reply:
> I don't think the name iotlab-ci is not very good.

So you think it's good ? So do I :)

Reply:
> > I don't think the name iotlab-ci is not very good.
> 
> So you think it's good ? So do I :)

:sweat_smile: ... fine let's not delay this PR on something that can be changed later :smiling_imp: :stuck_out_tongue_winking_eye:

Reply:
I agree with @miri64 on the name not being the most descriptive. I would like to see the word `tests` or `run-tests` in the name as well.